src/__tests__/utils.js

```diff
@@ -0,0 +1,15 @@
+import {deprecate, getDepreciationMessage} from '../utils'
+
+test('deprecate', () => {
+  const spy = jest.spyOn(console, 'warn').mockImplementation(() => {})
+
+  const messageWithReplacement = getDepreciationMessage('test', 'test')
```

Comment:
As I commented below, I think we should type out the full expected message here, rather than rebuilding it again with the function.